python/pdstools/plots/plot_base.py

```diff
@@ -672,12 +672,13 @@ def plotScoreDistribution(
             "BinPropensity",
             "ModelID",
         }
+
         df, _ = self._subset_data(table, required_columns, query)
         if modelids is not None:
             df = df.filter(pl.col("ModelID").is_in(modelids))
-        df = df.filter(pl.col("PredictorName") == "Classifier").collect()
+        df = df.filter(pl.col("PredictorName") != "Classifier").collect()
```

Comment:
Why flip this sign? I'm not sure if that looks right?

Reply:
Nice catch, that was a mistake, thought it was a different function.